src/client/post/Write/Write.js

```diff
@@ -256,7 +256,7 @@ class Write extends React.Component {
     const redirect = id !== this.draftId;
 
     this.props.saveDraft({ postData: data, id: this.draftId }, redirect);
-  }, 400);
+  }, 2000);
```

Comment:
I don't think this is a good solution. It's just changing numbers, in few days it might still cause problem, so we will increase it even more. At some point we will notice that it takes too long and we will decrease it again.
We can use flag that tells us we we are saving post or not, no need to come up with magic numbers.

Reply:
yeah youre right, ill update it to use a flag